src/type/directives.js

```diff
@@ -74,6 +75,12 @@ export class GraphQLDirective {
       });
     }
   }
+
+  getArgumentMap(): ObjMap<GraphQLArgument> {
```

Comment:
Not sure we want to add this to the `Directive` class. It should be fine for `findBreakingChanges` to just use the `this.args`, and just use `Array.find` when checking for the existence of an arg. I'm not too worried about performance given that there aren't usually many directives to begin with.

Reply:
that was my initial instinct. @mjmahone is that ok with you since you originally raised the concern?

lol @sam-swarr you are going to love what i did with `Schema::getDirectiveMap()` har har